src/main.ts

```diff
@@ -139,25 +139,20 @@ function insertBulkUI(): void {
 
 function toggleUI(): void {
   const editor = $(`#${SCRIPT_ID}_script_container`)!;
-  const active = editorOn();
-  editor.classList.toggle('hidden', active);
+  editor.classList.toggle('hidden');
+  const list = $('#imagelist-container, #imagelist_container');
   if (!active) {
```

Comment:
`active` is uninitialized on line 144.
I assume you meant to do `const active = !editor.classList.toggle('hidden')`? I still think the original is easier to reason about.